dashboard/api_test.py

```diff
@@ -309,6 +363,15 @@ def setUpTestData(cls):
             enr_end=now+timedelta(weeks=1),
             edx_key="course-v1:MITx+8.MechCX+2014_T1"
         )
+        # and a run that is past and verified
```

Comment:
I think the case you are describing is not possible in the current logic: if the course is current, we will return the current grade